### ISSUE TITLE
Implement PaCMAP

### ISSUE THREAD

**Issue by josevalim** [elixir-nx/scholar#238]
https://github.com/YingfanWang/PaCMAP

**Comment by krstopro** [comment]
@msluszniak Just to check - are you maybe working on this?

**Comment by msluszniak** [comment]
> @msluszniak Just to check - are you maybe working on this?

No, I don't. Currently, I will work on a notebook that will present the implemented methods ;)

**Comment by krstopro** [comment]
@msluszniak I see. I might take a shot at this one then.

**Comment by josevalim** [comment]
Btw, @krstopro, would you like to join the Scholar team alongside @msluszniak? No additional commitment is required from your side, it is mostly to recognize your contributions so far. :) Feel free to say no as well. Thank you. :)

**Comment by krstopro** [comment]
> Btw, @krstopro, would you like to join the Scholar team alongside @msluszniak? No additional commitment is required from your side, it is mostly to recognize your contributions so far. :) Feel free to say no as well. Thank you. :)

@josevalim @msluszniak If it's fine for you guys I'd be honored. :)

**Comment by msluszniak** [comment]
It's great to have you in Scholar team! :smiley:

**Comment by josevalim** [comment]
Invited. Welcome @krstopro!

**Comment by tiagodavi** [comment]
Hi, how are you folks organizing contributions here? "[help wanted]" is the high priority ?

**Comment by josevalim** [comment]
There is no priority per-se, whatever you feel is the most useful to you (or you are personally interested on).
